assets/javascripts/tests.js

```diff
@@ -184,26 +184,37 @@ function changeJobPrio(jobId, delta, linkElement) {
     });
 }
 
+function renderTestSummary(data) {
+    var html = ''
```

Comment:
Please use a semicolon here (for consistency).

Reply:
Won't be necessary, I joined the declaration and the initial value